console/main.go

```diff
@@ -0,0 +1,114 @@
+package console
```

Comment:
can we rename this file to something else instead of main.go. as there is no main func in this file

Reply:
+1